src/submission/migrations/0076_auto_20240402_1301.py

```diff
@@ -0,0 +1,38 @@
+# Generated by Django 3.2.20 on 2024-04-02 12:01
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+def migrate_relationship_to_fk(apps, schema_editor):
+    Article = apps.get_model('submission', 'Article')
+    for article in Article.objects.all():
+        for funder in article.funders.all():
+            print(f"Adding FK to {funder.pk}")
```

Comment:
Perhaps it would be best to `logger.debug` rather than polute stdout with these?